android/src/main/java/com/airbnb/android/react/maps/AirMapManager.java

```diff
@@ -165,6 +165,11 @@ public void setLoadingEnabled(AirMapView view, boolean loadingEnabled) {
         view.enableMapLoading(loadingEnabled);
     }
 
+    @ReactProp(name="moveOnMarkerPress", defaultBoolean = true)
+    public void setMoveOnMarkerPress(AirMapView view, boolean moveOnPress) {
```

Comment:
Just to clarify, should I add spaces around the `=` or remove it?  The other methods in the file use both standards.